Library/Formula/subversion.rb

```diff
@@ -156,6 +151,8 @@ def install
     if MacOS::CLT.installed?
       args << "--with-apr=/usr"
       args << "--with-apr-util=/usr"
+      # Ignore authz API bug in system httpd (fixed in httpd 2.4.16)
+      args << "--enable-broken-httpd-auth" if MacOS.version < :el_capitan
```

Comment:
One last thing: Does this need fixing for Homebrew's `apr` and `apr-util` as well?